cni/pkg/iptables/iptables_linux.go

```diff
@@ -56,7 +58,7 @@ func forEachInpodMarkIPRule(cfg *Config, f func(*netlink.Rule) error) error {
 		inpodMarkRule.Family = family
 		inpodMarkRule.Table = RouteTableInbound
 		inpodMarkRule.Mark = InpodTProxyMark
-		inpodMarkRule.Mask = InpodTProxyMask
+		inpodMarkRule.Mask = ptr.Of(uint32(InpodTProxyMask))
```

Comment:
What prompted this change?

Reply:
https://github.com/vishvananda/netlink/blob/b1ce50cfa9bea7652382b8c5ef6083f8d3f5f853/rule.go#L14

probably this